src/demux/readers/eos/NodeosActionReader.test.ts

```diff
@@ -151,6 +151,7 @@ describe("NodeosActionReader", () => {
       blockHash: "000f4241873a9aef0daefd47d8821495b6f61c4d1c73544419eb0ddc22a9e906",
       blockNumber: 20,
       previousBlockHash: "000f42401b5636c3c1d88f31fe0e503654091fb822b0ffe21c7d35837fc9f3d8",
+      timestamp: new Date("2018-06-16T05:59:49.500")
```

Comment:
Looks like it failed CI due to a missing trailing comma here. Will merge once that's fixed.